### DISCUSSION TITLE
谢谢大佬!请问考不考虑加入计时浮窗等功能呢?

### DISCUSSION THREAD

**Discussion by xuan0821** [Planshit/Tai#39] [General]
好希望这个能成为电脑版时间管理全能软件www

大佬辛苦了!!!谢谢大佬!

(以及:俺是UI设计www大佬的界面做的还蛮好看的啊!)
(如果以后你有需要也可以找我😋)

**Comment by noberumotto** [comment]
不太明白计时浮窗的意思，是指在当前使用的软件上添加一个浮窗计时吗？

（谢谢，别叫我大佬，菜鸟一个
